### PR TITLE
Add more display properties for tables

### PR DESCRIPTION
**Pull request by paulmolluzzo**
Adds more components being generated to make the list become:

* `View`
* `Block`
* `InlineBlock`
* `InlineListItem`
* `Flex`
* `Row`
* `Column`
* `Table`
* `InlineTable`
* `TableRowGroup`
* `TableHeaderGroup`
* `TableFooterGroup`
* `TableRow`
* `TableCell`
* `TableColumnGroup`
* `TableColumn`
* `TableCaption`

### CODE REVIEW

**Comment by paulmolluzzo** [comment]
@kentcdodds @smyte Any thoughts on this? It's more than `jsxstyle` is generating, but less than [the complete display property list](https://developer.mozilla.org/en-US/docs/Web/CSS/display).

Is there any reason we would not want all these `table`-related items included (even if they aren't used often)? Personally I'd use the actual element, like `glamorous.table()`, but maybe there's value for someone by including them.

File: index.js [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
 const InlineBlock = glamorous.div({display: 'inline-block'})
+const InlineListItem = glamorous.div({display: 'inline-list-item'})
+
+// Flex
 const Flex = glamorous.div({display: 'flex'})
 const Row = glamorous.div({display: 'flex', flexDirection: 'row'})
 const Column = glamorous.div({display: 'flex', flexDirection: 'column'})
 
+// Tables
+const Table = glamorous.div({display: 'table'})
```
Comment (kentcdodds) [inline index.js]: You could make all this more terse by adding `export` to every const declaration.

```js
export const Table = ...
```

Then you could drop the export at the bottom

**Review by kentcdodds — APPROVED** [review]
Looks great!

File: index.js (hunk 1 above)
Comment (paulmolluzzo) [reply index.js]: Sure could!

Comment (paulmolluzzo) [reply index.js]: Updated with your suggestion. 💯

File: rollup.config.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous-jsxstyle.es.js', format: 'es'},
         {dest: 'dist/glamorous-jsxstyle.cjs.js', format: 'cjs'}
   ],
-  exports: 'default',
+  exports: 'auto',
```
Comment (kentcdodds) [inline rollup.config.js:17]: Cool!

**Review by kentcdodds — APPROVED** [review]
Looks great to me!

**Pull request merged**